queries.sql

```diff
@@ -166,3 +166,8 @@ JOIN vets ON vets.id = visits.vet_id
 WHERE vets.name = 'Maisy Smith'
 GROUP BY species.name
 ORDER BY COUNT(animals.id) DESC LIMIT 1;
+
+-- Queries for performance measurement
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;
+EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
+EXPLAIN ANALYZE SELECT * FROM owners where email = 'owner_18327@mail.com';
```

Comment:
* Kindly improve the execution time for this query as the execution time on the PR screenshot can be lower

Reply:
By adding a specific index for the target vet, we achieved a performance improvement of 50%

![image](https://user-images.githubusercontent.com/80157011/229674091-2325e98c-5463-40f0-b9b9-55195e63e93a.png)